docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+
+.. _method_documentation:
+
+======
+Method
+======
+
+While **mrsimulator**'s organization of the :ref:`spin_sys_api` object and its
+composite objects, :ref:`site_api`, and :ref:`coupling_api` are easily
+understood by anyone familiar with the underlying physical concepts, the
+organization of the :ref:`method_api` object in **mrsimulator** and its related
+composite objects require a more detailed explanation of their design. This
+section assumes that you are already familiar with the topics covered in the
+introduction sections :ref:`getting_started`,
+:ref:`introduction_isotopomers_example`, and :ref:`fitting_example`.
+
+
+
+Overview
+--------
+
+An experimental NMR method involves a sequence of rf pulses, free evolution
+periods, and sample motion. The **Method** object in **mrsimulator** models the
+spectrum from an NMR pulse sequence. The **Method** object is designed to be
+versatile in its ability to model spectra from various multi-pulse NMR methods
+using concepts from the `symmetry pathway approach
+<https://doi.org/10.1016/j.pnmrs.2010.11.003>`_ where a pulse sequence is
+understood in terms of a set of desired (and undesired)  *transition pathways*.
+Each transition pathway is associated with a single resonance in a
+multi-dimensional NMR spectrum. The transition pathway signal encodes
+information about the spin system interactions in its amplitude and correlated
+frequencies. Consider the illustration of a 2D pulse sequence shown below, where
+a desired signal for the method is associated with a particular transition
+pathway, :math:`{\hat{A} \rightarrow \hat{B} \rightarrow \hat{C} \rightarrow
+\hat{D} \rightarrow \hat {E} \rightarrow \hat{F}}`.
+
+.. figure:: ../../_static/TransitionPathway.*
+    :width: 600
+    :alt: figure
+    :align: center
```

Comment:
@pjgrandinetti ``freq_contrib`` list can take a list of 0 to 9 **FrequencyEnum** objects. The UML diagram needs to be updated.

Also there is no **TotalMixQuery** object; its behavior should be discussed, but not included in UML